arch/xtensa/src/esp32s2/esp32s2_start.c

```diff
@@ -110,6 +111,9 @@ typedef enum
   CACHE_LINE_SIZE_64B = 2,            /* 64 Byte cache line size */
 } cache_line_size_t;
 
+#define CACHE_SIZE_8KB  CACHE_SIZE_HALF
+#define CACHE_SIZE_16KB CACHE_SIZE_FULL
```

Comment:
There is no need to create these definitions. Just another indirection that adds little to code comprehension.
The use of `cache_size_t` enum itens is already conditioned by ifdefs for 8KB and 16KB.